targets.json

```diff
@@ -113,6 +113,32 @@
                 "firmware": "Unified_ESP32_2400_RX"
             }
         },
+        "tx_dual": {
+            "nano": {
+                "product_name": "BAYCKRC 900/2400 Dual Band 1W Nano TX",
+                "lua_name": "B 9002G4 Nano TX",
+                "layout_file": "BAYCKRC Dual Band.json",
+                "upload_methods": ["uart", "wifi"],
+                "min_version": "3.5.2",
+                "platform": "esp32",
+                "firmware": "Unified_ESP32_LR1121_TX"
+            },
+            "nano_gemini": {
+                "product_name": "BAYCKRC 900/2400 Dual Band 1W Nano Gemini TX",
+                "lua_name": "B 9002G4 NanoGTX",
+                "layout_file": "BAYCKRC Dual Band.json",
+                "overlay": {
+                    "radio_busy_2": 39,
+                    "radio_dio1_2": 34,
+                    "radio_nss_2": 15,
+                    "radio_rst_2": 21
+                },
+                "upload_methods": ["uart", "wifi"],
+                "min_version": "3.4.0",
```

Comment:
Shouldn't that also be 3.5.2?

Reply:
This is fine as GemX functionality was added earlier